pymc/distributions/shape_utils.py

```diff
@@ -472,8 +472,8 @@ def convert_size(size: Size) -> Optional[StrongSize]:
     if size is None:
         return None
 
-    if isinstance(size, int) or (isinstance(size, TensorVariable) and size.ndim == 0):
-        size = (size,)
+    if isinstance(size, int) or (isinstance(size, TensorVariable) and size.ndim >= 1):
```

Comment:
I think size can never have `ndim >= 1`, so this check doesn't make sense. We need to make sure logic works for the two possible cases (ndim=0 and ndim=1)

Reply:
@ricardoV94 made a new edit making sure it only accepts ndim=0 and ndim=1 please have a look